RELEASE-NOTES.md

```diff
@@ -4,6 +4,70 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+- [@patternfly/react-catalog-view-extension: 1.4.5](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/1.4.5)
+- [@patternfly/react-charts: 5.3.3](https://www.npmjs.com/package/@patternfly/react-charts/v/5.3.3)
+- [@patternfly/react-core: 3.140.5](https://www.npmjs.com/package/@patternfly/react-core/v/3.140.5)
+- [@patternfly/react-inline-edit-extension: 2.17.5](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/2.17.5)
+- [@patternfly/react-styles: 3.7.2](https://www.npmjs.com/package/@patternfly/react-styles/v/3.7.2)
+- [@patternfly/react-table: 2.27.5](https://www.npmjs.com/package/@patternfly/react-table/v/2.27.5)
+- [@patternfly/react-tokens: 2.8.2](https://www.npmjs.com/package/@patternfly/react-tokens/v/2.8.2)
+- [@patternfly/react-topology: 2.14.5](https://www.npmjs.com/package/@patternfly/react-topology/v/2.14.5)
+- [@patternfly/react-virtualized-extension: 1.4.6](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/1.4.6)
+- [@patternfly/react-icons: 3.15.0](https://www.npmjs.com/package/@patternfly/react-icons/v/3.15.0)
+
+### Components:
+- **Context selector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
+- **Data toolbar:** Updated html structure to match core ([#3701](https://github.com/patternfly/patternfly-react/pull/3710))
+- **Filterable table:** Fixed the TextInput type in filter table demo ([#3652](https://github.com/patternfly/patternfly-react/pull/3652))
+- **Popover:** Added min-width override property ([#3601](https://github.com/patternfly/patternfly-react/pull/3601))
+- **Radio:** Added support for optional description ([#3621](https://github.com/patternfly/patternfly-react/pull/3621))
+- **Select:** Handled undefined default for checkbox select ([#3711](https://github.com/patternfly/patternfly-react/pull/3711))
+- **Simple list:** Added Simple list component ([#3645](https://github.com/patternfly/patternfly-react/pull/3645))
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
+- **Wizard:** Added typeto wizard context ([#3572](https://github.com/patternfly/patternfly-react/pull/3572))
+
+### Catalog view extension
+- **Catalog tile:** 
+  - Adjusted max width of header image ([#3628](https://github.com/patternfly/patternfly-react/pull/3628))
+  - Reduced padding between badge and logo ([#3644](https://github.com/patternfly/patternfly-react/pull/3644))
+  - Fixed styles that stopped working ([#3635](https://github.com/patternfly/patternfly-react/pull/3635))
+  - Fixed linting errors ([#3714](https://github.com/patternfly/patternfly-react/pull/3714))
+
+### Charts
+- **React charts:** Fixed linting errors ([#3713](https://github.com/patternfly/patternfly-react/pull/3713))
```

Comment:
I wonder if we should put this under "Linting" with all the other PRs that fixed linting issues?

Reply:
I agree - will do